app/components/feedback/edit_button_component.rb

```diff
@@ -1,14 +1,32 @@
 # frozen_string_literal: true
 
 class Feedback::EditButtonComponent < ViewComponent::Base
-  def initialize(feedback:, current_user:)
+  VARIANTS = {
+    button: 'btn-primary',
+    link: 'btn-link btn-xs sm:btn-sm hover:no-underline'
+  }.freeze
+
+  def initialize(feedback:, current_user:, style: :button, classes: '')
     @feedback = feedback
     @current_user = current_user
+    @style = style
+    @classes = classes
+  end
+
+  def call
+    variant = VARIANTS.fetch(style)
+
+    link_to(
+      edit_feedback_path(feedback),
+      class: "btn btn-sm capitalize #{variant} #{classes}"
+    ) do
+      content || 'Edit'
```

Comment:
where does the`content` come from?

Reply:
It's a ViewComponent thing. It comes from a block that's passed to the component. This is a way for the button to have different kinds of text. So if I wanted the button to say "Edit Feedback" instead of just edit ->
```rb
<%= render Feedback::EditButtonComponent(**args) do %>
  Edit Feedback
<% end %>
```
The `content` will be "Edit Feedback" and that's what will get rendered. With no block passed, `content` is `nil`, so I have it default to "Edit".

Reply:
gotcha! That is really cool!